system/Drivers/CMSIS/Device/ST/STM32H7xx/Source/Templates/system_stm32h7xx.c

```diff
@@ -93,29 +94,29 @@
 /* #define VECT_TAB_SRAM */
 #if defined(VECT_TAB_SRAM)
 #define VECT_TAB_BASE_ADDRESS   D2_AXISRAM_BASE   /*!< Vector Table base address field.
-                                                       This value must be a multiple of 0x200. */
+                                                       This value must be a multiple of 0x300. */
```

Comment:
0x300 should be reported to https://github.com/stm32duino/Arduino_Core_STM32/blob/cf51e49ae2094dafae86fcd26ddcd02d9b3056d6/system/STM32H7xx/system_stm32h7xx.c#L80-L99

Reply:
Right, nice catch. Done